src/util/isIdentifier.ts

```diff
@@ -46,6 +46,7 @@ const RESERVED_WORDS = new Set([
   "implements",
   "interface",
   "let",
+  "override",
```

Comment:
`override` is actually *not* a reserved word in JS or TS, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#keywords . It's a "contextual keyword", so it can be used as a variable name, but also has meaning in certain contexts. So it actually shouldn't be added to this file.